platform/AppStoreConnect/Sources/AppStoreConnectCMD/DistributeToBetaTesters.swift

```diff
@@ -0,0 +1,41 @@
+import ArgumentParser
+import Foundation
+
+struct DistributeToBetaTesters: ParsableCommand {
```

Comment:
Ah, I see, the terminology even made it here. So does Apple actually call those "beta groups"? I'd say they are "TestFlight groups", more accurately.